snapshots/who/2024-04-09/polio_historical.py

```diff
@@ -0,0 +1,31 @@
+"""Script to create a snapshot of dataset.
+
+The data is no longer available from the WHO but it is available on web archive e.g. here "https://web.archive.org/web/20200713223806/http://www.who.int/immunization/monitoring_surveillance/data/incidence_series.xls"
+
```

Comment:
WHO have a lot of problems with broken links and maintaining older documents and data, so I think it is likely because of this that the file is hard to access, rather than any particular issues with it. I will check with Saloni though.